crates/biome_js_analyze/src/lint/correctness/no_unused_variables.rs

```diff
@@ -26,9 +24,11 @@ declare_rule! {
     /// The pattern of having an underscore as prefix of a name of variable is a very diffuse
     /// pattern among programmers, and Biome decided to follow it.
     ///
-    /// This rule won't report unused imports.
+    /// This rule won't report unused imports or unused function parameters.
     /// If you want to report unused imports,
-    /// enable [noUnusedImports](https://biomejs.dev/linter/rules/no-unused-imports/).
+    /// enable [noUnusedImports](https://biomejs.dev/linter/rules/no-unused-imports/);
+    /// to report unused function parameters, enable
+    /// [noUnusedFunctionParameters](https://biomejs.dev/linter/rules/no-unused-function-parameters/)
```

Comment:
We can't change the documentation like this. This change will go straight to production and users will be misled. I think we should **add** a new paragraph like this:

```md
From `v1.9.0`, the rule won't check unused function parameters anymore, users should switch to [noUnusedFunctionParameters](https://biomejs.dev/linter/rules/no-unused-function-parameters/)
```

I wrote `1.9.0` because in `v1.8.0` the new rule is nursery, and we can't suggest a rule that is still in their incubation period.

Reply:
Updated docs as suggested